CHANGES.rst

```diff
@@ -43,6 +43,8 @@ Infrastructure, Utility and Other Changes and Additions
 
 - Versions of astropy <4 and numpy <1.16 are no longer supported. [#2163]
 
+- The package ``six`` is no longer a requirement. [#2165]
```

Comment:
this doesn't need a changelog as users wouldn't notice the cleanup.

Reply:
Users should not notice the changes in the code, but I still added a change log entry to make it clear that the installation requirements (which are obviously visible to the users) are now different.

Reply:
It wasn't mentioned before either when we stiched to standalone six rather than bundled, so no need to mention it.